tests/cpp/aot/python_scripts/comet_aot.py

```diff
@@ -40,11 +40,7 @@
 count = ti.field(ti.i32, ())
 img = ti.field(ti.f32, (res, res))
 
-sym_arr = ti.graph.Arg(ti.graph.ArgKind.NDARRAY,
-                       'arr',
-                       ti.f32,
-                       field_dim=3,
-                       element_shape=())
+sym_arr = ti.graph.Arg(ti.graph.ArgKind.NDARRAY, 'arr', ti.f32, ndim=3)
```

Comment:
nit: shall  we spread `dtype=ti.f32` across the tests to encourage users to specify kw for better clarity? ;)

Reply:
Have updated all occurrences with kw dtype inside tests and examples!